### PR TITLE
Moved functions from the core repo.

### LINKED ISSUE #39: Move creator utils from the core
Some of the utils we found there are so generic that can be moved here.

### PR DESCRIPTION
**Pull request by Reinmar**
Fixes #39.

### CODE REVIEW

File: tests/dom/setdatainelement.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+describe( 'setDataInElement', () => {
+	[ 'textarea', 'template', 'div' ].forEach( ( elementName ) => {
+		it( 'should set the content of a ' + elementName, () => {
+			const el = document.createElement( elementName );
+			const expectedData = '<b>foo</b>';
+
+			setDataInElement( el, expectedData );
+
+			const actualData = getDataFromElement( el );
+			expect( actualData ).to.equal( actualData );
```
Comment (pjasiun) [inline tests/dom/setdatainelement.js]: You expect `expect( actualData ).to.equal( actualData );` and it is true? It is unexpected!

File: tests/dom/getdatafromelement.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+ */
+
+/* bender-tags: dom, browser-only */
+
+'use strict';
+
+import getDataFromElement from '/ckeditor5/utils/dom/getdatafromelement.js';
+
+describe( 'getDataFromElement', () => {
+	[ 'textarea', 'template', 'div' ].forEach( ( elementName ) => {
```
Comment (pjasiun) [inline tests/dom/getdatafromelement.js]: `for( elementName of [ 'textarea', 'template', 'div' ] )` - is not it beautiful?

Comment (Reinmar) [inline tests/dom/getdatafromelement.js]: No, because you forgot a space.

**Comment by Reinmar** [comment]
I fixed the actual==actual test.

**Pull request merged**